### DISCUSSION TITLE
Wishlist: armv7 and aarch64 Linux builds

### DISCUSSION THREAD

**Discussion by tidux** [Orama-Interactive/Pixelorama#594] [Ideas]
**Feature/enhancement description:**
Pixelorama is already available on Flathub with an official package, which is great!  Adding armv7 and aarch64 (32 bit and 64 bit ARM) builds would make the Raspberry Pi or other ARM devices a solid choice for making pixel art.  It would even help with ChromeOS, since there are ChromeOS ARM devices that can run flatpaks.

**Comment by Pumpkinwaffle** [comment]
Did you try FRT?

https://github.com/efornara/frt

**Comment by chookity-pokk** [comment]
@Pumpkinwaffle  How would FRT help? I know it is basically Godot for ARM but how would one be able to run Pixelorama off of that?

**Comment by Pumpkinwaffle** [comment]
Well pixelorama is made in Godot. And as there is no official support for arm, FRT is your best shot at getting it to work.

Download the pixelorama source code, open it in Godot and use the export templates provided by FRT. I've gotten a quite a few godot projects running on my Linux arm handheld with it.

Depends on the device you want to run it on. I belive someone compiled the godot engine for raspberry pi as well. If FRT doesn't work you can always try that.

**Comment by chookity-pokk** [comment]
Interesting, thanks for the tip. I'll give that a go. If it happens to work I will try to ping the devs on it or I'll host an unofficial raspberry pi version. I noticed there was an unofficial godot version for raspberry pi but I also haven't used it yet. 

https://github.com/hiulit/Unofficial-Godot-Engine-Raspberry-Pi

**Comment by chookity-pokk** [comment]
I have an unofficial ARM build that I would love the devs to look at. https://gitlab.com/greenbeast/pixelorama-arm

**Comment by OverloadedOrama** [comment]
Good news, Pixelorama v1.0 will use Godot 4, which now has official export templates for ARM 32 and 64 bit. I plan on including these builds when the stable version of 1.0 is out, so I will close the proposal when that time comes.
